agent/config_test.go

```diff
@@ -107,8 +107,12 @@ func TestDecodeConfig(t *testing.T) {
 			c:  &Config{ACLMasterToken: "a"},
 		},
 		{
+			// Decode will enable ACL replication if the token is set.
```

Comment:
We usually don't add comments in config test cases. The test describes what happens.